plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/linked-product-control.tsx

```diff
@@ -136,35 +147,40 @@ const LinkedProductControl = ( {
 	location: WooCommerceBlockLocation;
 	usesReference: string[] | undefined;
 } ) => {
-	const REFERENCE_TYPE_PRODUCT = 'product';
-	const isProductReferenceAvailable =
-		location.type === REFERENCE_TYPE_PRODUCT;
+	const isProductLocation = location.type === 'product';
+	const isUsesReferenceIncludesProduct =
+		!! usesReference?.includes( 'product' );
+	const isCartLocation = location.type === 'cart';
+	const isUsesReferenceIncludesCart = !! usesReference?.includes( 'cart' );
```

Comment:
Same here; what do you think about renaming this const to `hasCartReference`?

Reply:
I don't have a strong opinion on this, so I made the changes in d6de3a2